.github/workflows/test-k6-build-publish-docker.yml

```diff
@@ -66,21 +66,22 @@ jobs:
           image-ref: ${{env.IMAGE-NAME}}:${{env.IMAGETAG}}
           exit-code: "1"
           severity: "CRITICAL,HIGH"
-      - name: "Login to Azure Container registery"
+      - name: 'Login to Azure Container registery'
         uses: azure/docker-login@v1
         with:
           login-server: ${{ secrets.REGISTRY_URL }}
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
+      - name: 'Build the tagged Docker image'
```

Comment:
må ha berre 1 bindestrek - elles blir det nytt step. sjå https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsteps. Du må sjekke syntax før du legg inn endringar!